packages/stark-core/src/common/translations/translations.ts

```diff
@@ -0,0 +1,23 @@
+import { TranslateService } from "@ngx-translate/core";
+const _cloneDeep: Function = require("lodash/cloneDeep");
+
+import { StarkLocale } from "./locale.intf";
+import { translationsEn } from "./translations/en";
+import { translationsFr } from "./translations/fr";
+import { translationsNl } from "./translations/nl";
+
+export function mergeTranslations(translateService: TranslateService, ...args: StarkLocale[]): void {
```

Comment:
Could you maybe add a JSDoc for this function to describe what it actually does and why?